pavelib/utils/envs.py

```diff
@@ -82,104 +70,22 @@ class Env:
+    PRINT_SETTINGS_LOG_FILE = GEN_LOG_DIR / "print_settings.log"
 
     # Detect if in a Docker container, and if so which one
-    SERVER_HOST = os.environ.get('BOK_CHOY_HOSTNAME', '0.0.0.0')
-    USING_DOCKER = SERVER_HOST != '0.0.0.0'
-    SETTINGS = 'bok_choy_docker' if USING_DOCKER else 'bok_choy'
-    DEVSTACK_SETTINGS = 'devstack_docker' if USING_DOCKER else 'devstack'
+    # USING_DOCKER = SERVER_HOST != '0.0.0.0'
+    DEVSTACK_SETTINGS = 'devstack'
     TEST_SETTINGS = 'test'
 
-    BOK_CHOY_SERVERS = {
-        'lms': {
-            'host': SERVER_HOST,
-            'port': os.environ.get('BOK_CHOY_LMS_PORT', '8003'),
-            'log': BOK_CHOY_LOG_DIR / "bok_choy_lms.log"
-        },
-        'cms': {
-            'host': SERVER_HOST,
-            'port': os.environ.get('BOK_CHOY_CMS_PORT', '8031'),
-            'log': BOK_CHOY_LOG_DIR / "bok_choy_studio.log"
-        }
-    }
-
-    BOK_CHOY_STUBS = {
-
-        'xqueue': {
-            'port': 8040,
-            'log': BOK_CHOY_LOG_DIR / "bok_choy_xqueue.log",
-            'config': 'register_submission_url=http://0.0.0.0:8041/test/register_submission',
-        },
-
-        'ora': {
-            'port': 8041,
-            'log': BOK_CHOY_LOG_DIR / "bok_choy_ora.log",
-            'config': '',
-        },
-
-        'comments': {
-            'port': 4567,
-            'log': BOK_CHOY_LOG_DIR / "bok_choy_comments.log",
-        },
-
-        'video': {
-            'port': 8777,
-            'log': BOK_CHOY_LOG_DIR / "bok_choy_video_sources.log",
-            'config': f"root_dir={VIDEO_SOURCE_DIR}",
-        },
-
-        'youtube': {
-            'port': 9080,
-            'log': BOK_CHOY_LOG_DIR / "bok_choy_youtube.log",
-        },
-
-        'edxnotes': {
-            'port': 8042,
-            'log': BOK_CHOY_LOG_DIR / "bok_choy_edxnotes.log",
-        },
-
-        'ecommerce': {
-            'port': 8043,
-            'log': BOK_CHOY_LOG_DIR / "bok_choy_ecommerce.log",
-        },
-
-        'catalog': {
-            'port': 8091,
-            'log': BOK_CHOY_LOG_DIR / "bok_choy_catalog.log",
-        },
-
-        'lti': {
-            'port': 8765,
-            'log': BOK_CHOY_LOG_DIR / "bok_choy_lti.log",
-        },
-    }
-
-    # Mongo databases that will be dropped before/after the tests run
-    MONGO_HOST = 'edx.devstack.mongo' if USING_DOCKER else 'localhost'
-    BOK_CHOY_MONGO_DATABASE = "test"
-    BOK_CHOY_CACHE_HOST = 'edx.devstack.memcached' if USING_DOCKER else '0.0.0.0'
-    BOK_CHOY_CACHE = memcache.Client([f'{BOK_CHOY_CACHE_HOST}:11211'], debug=0)
-
     # Test Ids Directory
     TEST_DIR = REPO_ROOT / ".testids"
 
     # Configured browser to use for the js test suites
-    SELENIUM_BROWSER = os.environ.get('SELENIUM_BROWSER', 'firefox')
-    if USING_DOCKER:
-        KARMA_BROWSER = 'ChromeDocker' if SELENIUM_BROWSER == 'chrome' else 'FirefoxDocker'
-    else:
-        KARMA_BROWSER = 'FirefoxNoUpdates'
+    KARMA_BROWSER = 'FirefoxNoUpdates'
```

Comment:
I think you'll need to leave this logic in so that people can choose chrome or firefox won't you?  ie. if someone sets `SELENIUM_BROWSER` to `chrome`, we'd want KARMA to do the same.

Reply:
@salman2013 did you look at this? did you reach a conclusion?

Reply:
@salman2013 and I talked about this offline, and we'll leave these variables as is for now because they're being used by devstack.  We'll rename this variable in devstack first and then come back and update it here.